blocks.md

```diff
@@ -771,6 +771,7 @@ Creates a block which displays network connection information from NetworkManage
 [[block]]
 block = "networkmanager"
 on_click = "alacritty -e nmtui"
+interface_name_filters = ["br\\-[0-9a-f]{12}", "docker\\d+"]
```

Comment:
maybe call this `interface_name_exclude`, so that later we could add `interface_name_include` as well?

this way the option name is also self-explanatory, which is a plus :)

Reply:
To give a bit of background - I also want to hide Docker interfaces, but really, I could go about it in a simpler way - by using an "include" pattern to only match ethernet and wireless interfaces.

Reply:
Here's another idea - why have a list of patterns, when youcould have `"foo|bar"` instead of `["foo", "bar"]`?

Reply:
Thanks for taking your time to review

> maybe call this `interface_name_exclude`, so that later we could add `interface_name_include` as well?
> 
> this way the option name is also self-explanatory, which is a plus :)

Yeah that name could definitely be improved. It's very ambiguous as it is right now. Will change it.

> To give a bit of background - I also want to hide Docker interfaces, but really, I could go about it in a simpler way - by using an "include" pattern to only match ethernet and wireless interfaces.

Makes sense. I'll add an include option as well

> Here's another idea - why have a list of patterns, when you could have `"foo|bar"` instead of `["foo", "bar"]`?

I'm not really good with regex so I'm not sure this excuse is valid - wouldn't forcing the user to rely on `|` for multiple patterns might make stuff like regex backreferences awkward? Or in other words, is merging a list of arbitrary regex patterns always as easy as adding `|` between them? If not, I think it's a good excuse to keep the list

Reply:
I'm honestly not sure regarding backreferences. I'm used to Go's regular expressions, which don't have those, so it's generally safe to join many regexes with `|` as long as the intended behavior is only "does it match or not".

This is probably a judgement call when it comes to the config UX and how it will be used in the future :) I think either method is reasonable, I just wanted to flag what could be a simpler way to go about this.

Reply:
I just did some testing with back-references, I could come up with rather simple patterns where attempting to combine them would require further modifications to the patterns themselves (i.e it's not always as simple as `|`), so I I'll keep the list option.

Reply:
Added include filters, renamed config as requested. Will now only compile each regexp once.